packages/shared/src/lib/analytics.ts

```diff
@@ -93,6 +108,8 @@ export enum TargetType {
   LoginButton = 'login button',
   SignupButton = 'signup button',
   SquadJoinButton = 'squad join button',
+  SearchRecommendation = 'search rec',
+  SearchSource = 'search source',
```

Comment:
I think this ones implementation is missing

Reply:
good catch 👍